src/base/BaseTool.js

```diff
@@ -111,16 +111,35 @@ export default class BaseTool {
   /**
    * Applys the requested mixins to the class.
    *
-   * @param  {Array} mixinArray An array of mixin names (strings).
+   * @param  {Array} mixinArray An array of mixin URIs (strings).
    */
   _applyMixins (mixins) {
+    // TODO:
+    // TODO: deal with: missing namespace + type
+
+    if (typeof cToolsImport !== 'function') { // Makes testing a lot easier.
+      return;
+    }
```

Comment:
It allows you to use `BaseTool` without mocking it, otherwise tools try to instantiate `mixin`s that may not be registered to the library yet. I'd be happy to discuss any ideas you'd have for a better approach, or some jest magic that makes this unnecessary.